src/src/radio.c

```diff
@@ -2592,7 +2592,7 @@ static void updateRxFrameDetails(RAIL_RxPacketDetails_t *pPacketDetails,
         sReceiveAckFrame.mInfo.mRxInfo.mLqi       = pPacketDetails->lqi;
         sReceiveAckFrame.mInfo.mRxInfo.mTimestamp = pPacketDetails->timeReceived.packetTime;
 #if OPENTHREAD_RADIO && OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE == 1
-        sReceiveAckFrame.mIid                     = iid;
+        sReceiveAckFrame.mIid = iid;
```

Comment:
Is make pretty working as expected? Looks like `=` does not line up..

Reply:
Yeah, this matches what I would expect. Alignment of equals chars is being reset by the `#if`